aep/general/0131/aep.md.j2

```diff
@@ -10,8 +10,9 @@ resource.
 
 ## Guidance
 
-APIs **must** provide a get method for resources. The purpose of the get method
+ - APIs **must** provide a get method for resources. The purpose of the get method
 is to return data from a single resource.
+ - Some resources take longer to be retrieved than is reasonable for a regular API request. In this situation, the API should use a long-running operation (AEP-151).
```

Comment:
```suggestion
 - Some resources take longer to be retrieved than is reasonable for a regular API request. In this situation, the API should use a [long-running operation](/long-running-operations).
```

Generally we should not use the hard-coded number, since we may repoint the AEP to a new number based on a major revision.

Also we have a convention of linking vs calling out the specific one.